docs/HowToGuides/GettingStarted.md

```diff
@@ -357,7 +357,7 @@ while retaining the option of building with Ninja on the command line.
 
 Assuming that you have already [built the toolchain via Ninja](#the-actual-build),
 several more steps are necessary to set up this environment:
-* Generate Xcode projects with `utils/build-script --release --swift-darwin-supported-archs "$(uname -m)" --xcode --clean`.
+* Generate Xcode projects with `utils/build-script --swift-darwin-supported-archs "$(uname -m)" --xcode --clean`.
```

Comment:
Do you know why we need to pass this flag?

```
--swift-darwin-supported-archs
```

@edymtt might have more insight.

Reply:
Yes, because of errors like this one:
```
CMake Error in SwiftCompilerSources/CMakeLists.txt:
The custom command generating

  /Users/mac/Desktop/swift-project/build/Xcode-DebugAssert/swift-macosx-x86_64/$(CONFIGURATION)$(EFFECTIVE_PLATFORM_NAME)/lib/swift/macosx/x86_64/libcxxshim.modulemap

is attached to multiple targets:

  lib-swift-macosx-arm64-libcxxshim.modulemap
  lib-swift-macosx-x86_64-libcxxshim.modulemap

but none of these is a common dependency of the other(s).  This is not
allowed by the Xcode "new build system".

multiple targets not supported by Xcode
```